### ISSUE TITLE
error

### ISSUE THREAD

**Issue by HeliaHashemipour** [shaoyanpan/2D-Medical-Denoising-Diffusion-Probabilistic-Model-#10]
I don't want to use ResizeWithPadOrCropd but when I delete it I have this error:

<torch.utils.data.dataloader.DataLoader object at 0x7b8079a32da0>
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
[<ipython-input-84-06cda4d29b52>](https://localhost:8080/#) in <cell line: 23>()
     24     print('Epoch:', epoch)
     25     start_time = time.time()
---> 26     average_loss = train(model, optimizer, train_loader1, train_loss_history)
     27     print('Execution time:', '{:5.2f}'.format(time.time() - start_time), 'seconds')
     28     if epoch % 10 == 0:

8 frames
[/usr/local/lib/python3.10/dist-packages/torch/_tensor.py](https://localhost:8080/#) in __torch_function__(cls, func, types, args, kwargs)
   1277 
   1278         with _C.DisableTorchFunction():
-> 1279             ret = func(*args, **kwargs)
   1280             if func in get_default_nowrap_functions():
   1281                 return ret

RuntimeError: stack expects each tensor to be equal size, but got [1, 1468, 993] at entry 0 and [1, 1244, 1005] at entry 1

**Comment by shaoyanpan** [comment]
Right. Because if you delete that, you different samples (like different patients) have different sizes. So in our framework (usually also in most of other framework) does not accept the inconsistent size across different samples.
